File: matlab/example_cpf.m
% Add subfolders to PATH variable (can be deleted if package is already in
% folder within PATH or files are in the same catalog as this script).
addpath(genpath(join([fileparts(which(mfilename)),"\mHT Package"],"")));

% Specifing display format output to double precision
format long;

% Example parameters for the CPF functions.
x = 1; % Dimensionless
y = 1; % Dimensionless

% Displaying the cpf function outputs
disp('The output of the cpf_accurate function:');
disp(cpf_accurate(x,y));
disp('The output of the cpf_fast function:');
disp(cpf_fast(x,y));

File: python/example_cpf.py
try:
  from mHT.CPF import cpf_fast, cpf_accurate
except ImportError as msg: 
  raise SystemExit (str(msg) + '\nexample_cpf.py: mHT not found. Make sure catalog with mHT module is in same catalog as running script and is named "mHT"!')

# Example parameters for the cpf functions
x=1 # Dimensionless
y=1 # Dimensionless

print("The output of the cpf_accurate function:")
print(cpf_accurate(x,y))

print("The output of the cpf_fast function:")
print(cpf_fast(x,y))
